src/sim65/6502.c

```diff
@@ -205,7 +213,22 @@ int PrintCycles;
     unsigned Addr;                                              \
     Cycles = 5;                                                 \
     ZPAddr = MemReadByte (Regs.PC+1);                           \
-    Addr = MemReadZPWord (ZPAddr) + Regs.YR;                    \
+    Addr = MemReadZPWord (ZPAddr);                              \
+    if (PAGE_CROSS (Addr, Regs.YR))                             \
+        ++Cycles;                                               \
```

Comment:
The style in this file is to have curly brackets even for single instruction if-bodies.